src/it/echo/postbuild.groovy

```diff
@@ -18,7 +18,6 @@ String [] buildLog = new File(basedir, 'build.log')
 
 messageLines = buildLog.grep(~/^\[INFO\] Good Morning Mr Bond\.$/)
 
-assert messageLines.size() == 1: 'Configured message was expected in build log at INFO level'
+assert messageLines.size() == 1: 'Configured message was expected in build log at INFO level. Build log:' + buildLog
 
```

Comment:
I think it would be better to use https://maven.apache.org/plugins/maven-invoker-plugin/run-mojo.html#streamLogsOnFailures instead.